code/modules/projectiles/projectile/magic.dm

```diff
@@ -214,11 +214,6 @@ proc/wabbajack(mob/living/M)
 					var/datum/preferences/A = new()	//Randomize appearance for the human
 					A.copy_to(new_mob)
 
-					var/mob/living/carbon/human/human/H = new_mob
```

Comment:
There is no created proc to set a species as random, that would need to get created from scratch.